README.md

```diff
@@ -1,5 +1,21 @@
 # Large Scale Entity Linking
 
+## Description
+
+### Architecture
+
+![image](https://github.com/WDPS-Team/2019-WDPS/blob/architecture/docs/architecture/architectue2.0.png)
+
+- This project is to perform Entity Linking on a collection of web pages using entities from Freebase. 
+- The **Input** is a set of pages in WARC format, stored information that crawled from web. The final goal is to link the 'entity mention' in WARCfile to 'entities' in Freebase, and **Output** format is : [Document ID, Entity Mention, Freebase ID] 
+- This project consists of following steps: 
+    1. WARCSplitReader - Reading and splitting the WARCfile into lines and proccessing in parallel. Filtering the invalid data. 
+    2. TextExtraction - Cleaning the result from the last step, and then using `beautifulsoup` to extract title, main text  from webpages. And finally reformat the result as split the text into sentences. 
+    3. EntityRecognition - Processing the sentences `spacy`. Select the entities from text with certain types. We only interedted in this three types: `PERSON`,`ORGANIZATION` and `LOCATION`.  
```

Comment:
We are actually including all the entity types recognized by Spacy unless it is a number or a date. :)